ONSdigital/rm-collection-exercise-service
# Remove bi/always create b case at execution

# Motivation and Context

This change is needed to always create B cases when collection exercises are executed. 

# What has changed

Now the sample unit is always published to case to create a B case.



# How to test?

Tested with case and case-api changes execute a second collection exercise and check B cases are being created as well as BI cases when businesses have respondents enrolled for the survey.



# Links

[Trello] https://trello.com/c/ZWg5l0Kt


Linked PRs:
[Case API] https://github.com/ONSdigital/rm-casesvc-api/pull/24
[Case Service]  https://github.com/ONSdigital/rm-case-service/pull/56
